db_task2/electives.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+Include 'index.php';
+
+$regex='/[0-9]+/';
+$name=basename($_SERVER['REQUEST_URI']);
+$fileName='/electives.php/';
+if (!preg_match($regex, $name) && !preg_match($fileName, $name))
+{
+    die("Enter valid path param");
+}
+$electiveDictionary = getElectiveByID($name);
+$checkIfExists=$electiveDictionary["title"];
```

Comment:
`checkIfExists` - can you thing of a better name of the variable? Something that describes what the variable holds?